packages/webapp/package.js

```diff
@@ -1,49 +1,62 @@
 Package.describe({
   summary: "Serves a Meteor app over HTTP",
-  version: '1.4.0'
+  version: "2.0.0",
 });
 
-Npm.depends({connect: "2.30.2",
-             parseurl: "1.3.0",
-             send: "0.13.0",
-             useragent: "2.0.7"});
+Npm.depends({
+  connect: "3.6.5",
+  compression: "1.7.1",
+  errorhandler: "1.5.0",
+  "lodash.find": "4.6.0",
+  parseurl: "1.3.2",
+  send: "0.16.1",
+  "qs-middleware": "1.0.3",
+  useragent: "2.2.1",
+});
 
 Npm.strip({
   multiparty: ["test/"],
-  useragent: ["test/"]
+  useragent: ["test/"],
 });
 
 Cordova.depends({
-  'cordova-plugin-whitelist': '1.3.2',
-  'cordova-plugin-wkwebview-engine': '1.1.3',
-  'cordova-plugin-meteor-webapp': '1.4.2'
+  "cordova-plugin-whitelist": "1.3.2",
+  "cordova-plugin-wkwebview-engine": "1.1.3",
+  "cordova-plugin-meteor-webapp": "1.4.2",
 });
 
-Package.onUse(function (api) {
-  api.use('ecmascript');
-  api.use(['logging', 'underscore', 'routepolicy', 'boilerplate-generator',
-           'webapp-hashing'], 'server');
-  api.use(['underscore'], 'client');
+Package.onUse(function(api) {
+  api.use("ecmascript");
+  api.use(
+    [
+      "logging",
+      "underscore",
+      "routepolicy",
+      "boilerplate-generator",
+      "webapp-hashing",
+    ],
+    "server"
+  );
 
   // At response serving time, webapp uses browser-policy if it is loaded. If
   // browser-policy is loaded, then it must be loaded after webapp
   // (browser-policy depends on webapp). So we don't explicitly depend in any
   // way on browser-policy here, but we use it when it is loaded, and it can be
   // loaded after webapp.
-  api.mainModule('webapp_server.js', 'server');
-  api.export('WebApp', 'server');
-  api.export('WebAppInternals', 'server');
-  api.export('main', 'server');
+  api.mainModule("webapp_server.js", "server");
+  api.export("WebApp", "server");
+  api.export("WebAppInternals", "server");
+  api.export("main", "server");
 
-  api.mainModule('webapp_client.js', 'client');
-  api.export('WebApp', 'client');
+  api.mainModule("webapp_client.js", "client");
+  api.export("WebApp", "client");
 
-  api.mainModule('webapp_cordova.js', 'web.cordova');
+  api.mainModule("webapp_cordova.js", "web.cordova");
 });
 
-Package.onTest(function (api) {
-  api.use(['tinytest', 'ecmascript', 'webapp', 'http', 'underscore']);
-  api.addFiles('webapp_tests.js', 'server');
-  api.addFiles('webapp_client_tests.js', 'client');
-  api.addFiles('socket_file_tests.js', 'server');
+Package.onTest(function(api) {
+  api.use(["tinytest", "ecmascript", "webapp", "http", "underscore"]);
```

Comment:
Not opposed to most of the `prettier` changes, but any possibility to get them in a commit by themselves, just prior to the other changes? :smile:

Reply:
Yeah, I know it may be hard to un-prettier the code now, but please please don't mix a reformatting commit together with actual changes. Would it be possible to split the commits now?

Reply:
@benjamn I'll just revert and reapply the technical changes (they are small) and disable prettier for working on the meteor code base 👍